packages/firestore/src/local/indexeddb_schema.ts

```diff
@@ -131,7 +87,14 @@ export class DbMutationQueue {
      * After sending this token, earlier tokens may not be used anymore so
      * only a single stream token is retained.
      */
-    public lastStreamToken: string
+    public lastStreamToken: string,
+    /**
```

Comment:
newline

Reply:
I added newlines here and everywhere else where they were missing. I also tried to make this semi-valid JSDoc along with it (I still don't think it's valid as we would probably have to move the JSDoc comments to the function header).

Reply:
Note that prettier stripped the newlines again.

Reply:
Even if it is not for external documentation, it could show up in code completion when done properly - whereas right not it just shows up as a warning due do the `/** JSDoc */` marks.

I agree that this is mostly a non-issue though and happily pressed some buttons to revert this.